imap_data_access/cli.py

```diff
@@ -182,6 +230,14 @@ def main():
     query_parser.add_argument(
         "--extension", type=str, required=False, help="File extension (cdf, pkts, etc.)"
     )
+    query_parser.add_argument(
+        "--output-format",
+        type=str,
+        required=False,
+        help="How to format the output, default is 'table'",
+        choices=["table", "json"],
```

Comment:
Somehow I have never seen this `choices` parameter before. That's neat. Does this automatically raise an error if a user supplies something that is not in this list?

Reply:
Yep, and the help even gives the list of options.
```
imap-data-access query --instrument abc
usage: imap-data-access query [-h]
                              [--instrument {codice,glows,hi,hit,idex,lo,mag,swapi,swe,ultra}]
                              [--data-level DATA_LEVEL]
                              [--descriptor DESCRIPTOR]
                              [--start-date START_DATE] [--end-date END_DATE]
                              [--version VERSION] [--extension EXTENSION]
                              [--output-format {table,json}]
imap-data-access query: error: argument --instrument: invalid choice: 'abc' (choose from 'codice', 'glows', 'hi', 'hit', 'idex', 'lo', 'mag', 'swapi', 'swe', 'ultra')
```